src/app/data-planes/views/DataPlaneListView.vue

```diff
@@ -1,157 +1,89 @@
 <template>
-  <RouteView>
-    <RouteTitle
-      :title="t(`${props.isGatewayView ? 'gateways' : 'data-planes'}.routes.items.title`)"
-    />
-    <AppView>
-      <DataPlaneList
-        :data-plane-overviews="dataPlaneOverviews"
-        :is-loading="isLoading"
-        :error="error"
-        :next-url="nextUrl"
-        :page-offset="pageOffset"
-        :selected-dpp-name="props.selectedDppName"
-        :is-gateway-view="props.isGatewayView"
-        :gateway-type="props.gatewayType"
-        :dpp-filter-fields="dppFilterFields"
-        @load-data="loadData"
-      />
-    </AppView>
+  <RouteView
+    v-slot="{ route }"
+  >
+    <DataSource
+      v-slot="{data, error}: DataPlaneCollectionSource"
+      :src="`/${props.mesh}/dataplanes?page=${props.page}&size=${size}&search=${props.search}`"
+    >
+      <AppView>
+        <template #title>
+          <h2>
+            <RouteTitle
+              :title="t('data-planes.routes.items.title')"
+              :render="true"
+            />
+          </h2>
+        </template>
+        <KCard>
+          <template #body>
+            <DataPlaneList
+              data-testid="data-overview-table"
```

Comment:
**Suggestion**: So the column style suggestion works and to be consistent with the other list views.

```suggestion
              class="data-plane-proxy-collection"
              data-testid="data-plane-proxy-collection"
```